lib/matplotlib/table.py

```diff
@@ -57,24 +60,40 @@ class Cell(Rectangle):
     fontproperties : dict
         A dict defining the font properties of the text. Supported keys and
         values are the keyword arguments accepted by `.FontProperties`.
+    visible_edges : str
+        A substring of 'BRTL' (bottom, right, top, left) or one
+        of {'left', 'center', 'right'}
```

Comment:
```suggestion
        of  {'open', 'closed', 'horizontal', 'vertical'}.
```
? That's what `_edge_aliases` says.

Reply:
I'm wondering if it's worth to rethink rethink the parameter name and values. Users will have to touch their code anyway to move from `CustomCell` to `Cell`.

Comparing with HTML tables `border` would be a more common name than `visible_edges`.
For the values:
- For single edges the long forms 'left', 'right', 'bottom' and 'top' are more readable. `Cell(border='bottom')` vs. `Cell(visible_edges='B')`.
- 'horizontal' and 'vertical' may stay as is.
- 'open', 'closed' could better be described by 'none', 'all'.
- For arbitrary combinations, one could either keep `Cell(border='BR')` or accept long forms `Cell(border=['bottom', 'right'])` or `Cell(border=['bottom', 'right'])`.

Open for discussion. Not 100% sure if it's worth the effort, but if we want to change it, now would be the time.

Reply:
I suspect code actually referencing these CustomCell's will be pretty rare if non-existant -- also pre-2015 code will use Cell not CustomCell -- so with luck this will break very little code.

99% of table use is via the table.table function -- and this simply 'edges' as the keyword.

While I am here, I have another fix related to cell edges, it allows you to specify the cell edge color -- I think it might make sense to include that here.....  see 9196fa6442e9

Reply:
Re: bottom, left, right, top etc -- might be simplest just to add them to the list of _edge_aliases, along with 'none' and 'all', that way we don't have to deal with the value being either a string or a list. see 3c14d5b

Most reasonable cases then have a long form.

oh and I just noticed, unless all cell edges are drawn the specifying cell colors is pointless as the cell has no interior!